rabbitmq_amqp_python_client/qpid/proton/_transport.py

```diff
@@ -826,7 +826,7 @@ def __init__(self, mode: int) -> None:
     def _check(self, err: int) -> int:
         if err < 0:
             exc = EXCEPTIONS.get(err, SSLException)
-            raise exc("SSL failure.")
+            raise exc("SSL failure.", err)
```

Comment:
This was worth its weight in gold during debugging.
https://github.com/apache/qpid-proton/blob/main/c/src/ssl/schannel.cpp#L1967 resulting in an qpid proton`Timeout` instance is misleading. But at least the string mentions `SSL`.